sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java

```diff
@@ -419,18 +420,19 @@ public EnqueuedMessage enqueueMessage(String messageText) {
      *
      * <p>Receive a message</p>
      *
-     * {@codesnippet com.azure.storage.queue.queueClient.getMessages}
+     * {@codesnippet com.azure.storage.queue.queueClient.receiveMessages}
      *
      * <p>For more information, see the
      * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/get-messages">Azure Docs</a>.</p>
      *
-     * @return The first {@link DequeuedMessage} in the queue, it contains {@link DequeuedMessage#getMessageId()
-     * messageId} and {@link DequeuedMessage#getPopReceipt() popReceipt} used to interact with the message, additionally
-     * it contains other metadata about the message.
+     * @return The first {@link QueueMessageItem MessageItem} in the queue, it contains
+     * {@link QueueMessageItem#getMessageId() messageId} and
+     * {@link QueueMessageItem#getPopReceipt() popReceipt} used to interact with the message,
+     * additionally it contains other metadata about the message.
      * @throws QueueStorageException If the queue doesn't exist
      */
-    public PagedIterable<DequeuedMessage> getMessages() {
-        return getMessages(1, Duration.ofSeconds(30), null, Context.NONE);
+    public PagedIterable<QueueMessageItem> receiveMessage() {
```

Comment:
Similar comment to it's async counterpart. Should this just return `QueuedMessageItem` instead of `PagedIterable`?